wdl-grammar/src/v1.rs

```diff
@@ -21,6 +21,7 @@
 //! | `no_curly_commands`     | `v1::W002` | Pedantic    | [Link](lint::NoCurlyCommands)     |
 //! | `mixed_indentation`     | `v1::W004` | Style       | [Link](lint::MixedIndentation)    |
 //! | `missing_runtime_block` | `v1::W005` |Completeness | [Link](lint::MissingRuntimeBlock) |
+//! | `snake_case`            | `v1::W006` | Naming      | [Link](lint::SnakeCase)        |
```

Comment:
It's hard to see in GitHub, but I'd just like the last pipe align with the others in the table.

Reply:
> It's hard to see in GitHub

@adthrasher how do you feel about maintaining space aligned elements within your code? @claymcleod is obviously "pro", I am more "anti".

I find these a pain to maintain without the right IDE plugins (evidence in how Clay's "suggested change" got rendered by GH). I prefer a minimalist editing set-up, and would prefer we adopt styles that don't make us overly reliant on tailored IDE configurations. That makes me prefer "compact" table formatting:
```suggestion
//! | `snake_case` | `v1::W006` | Naming  | [Link](lint::SnakeCase) |
```
Ultimately I'll conform to whichever, but making my preference known.

Reply:
I don't have a strong preference. Space aligned looks nice, but as you (@a-frantz) note it can be annoying to maintain. Note that line 23 actually has incorrect spacing (`|Completeness` vs. `| Completeness`), which may reinforce the idea that these are hard to do correctly and maintain over time.

Reply:
> I find these a pain to maintain without the right IDE plugins (evidence in how Clay's "suggested change" got rendered by GH). I prefer a minimalist editing set-up, and would prefer we adopt styles that don't make us overly reliant on tailored IDE configurations.

You can convince me of this idea in general—especially for large, complicated tables. That said, in this particular instance, I have a hard time buying that this is a huge pain. Even without an IDE plugin, it's pretty straightforward to just add the spaces to make sure things align manually without a plugin.

My thought is this: if they end product is a rendered marked table (such as in a Markdown file), then I'm more okay with not formatting it in the code directly. Here, a significant part of the value is looking at the table within the docstring itself, so I feel more strongly it should be aligned in the text.

Reply:
Ok, that seems like a balanced approach. I'll conform to space-aligned in this and similar contexts 👍